python/pyspark/sql/tests/test_functions.py

```diff
@@ -157,6 +159,32 @@ def test_inverse_trig_functions(self):
             for c in cols:
                 self.assertIn(f"{alias}(a)", repr(f(c)))
 
+    def test_reciprocal_trig_functions(self):
+        # SPARK-36683: Add new built-in SQL functions: SEC and CSC
+        list = [0.0, math.pi/6, math.pi/4, math.pi/3, math.pi/2,
+                math.pi, 3 * math.pi / 2, 2 * math.pi]
+
+        df = self.spark.createDataFrame(list, types.DoubleType())
+
+        def to_reciprocal_trig(func):
+            return [1.0 / func(i) if func(i) != 0 else math.inf for i in list]
+
+        def get_values(l):
```

Comment:
`get_values` and `assert_close` are almost the same as ones defined in `test_math_functions`.

https://github.com/apache/spark/blob/adbea252db0f61ed3aaa45ea20b704bad6c47408/python/pyspark/sql/tests/test_functions.py#L114-L120

Can we move them to `testing/sqlutils.py` to avoid duplication?
@HyukjinKwon What do you think?

Reply:
Sure, I think we can.